### PR TITLE
5490 content change

### PR DESCRIPTION
**Pull request by cehsu**
Connects to department-of-veterans-affairs/vets.gov-team#2867

Intro page:
Boldface the first line in blue box: You won’t be able to save your work or come back to finish.

screen shot 2017-05-28 at 4 04 38 pm

 Capitalize B in "Apply for benefits"

screen shot 2017-05-28 at 3 42 39 pm

Veteran info page:

 Add this to the top of the page:
You aren’t required to fill in all fields, but VA can evaluate your claim faster if you provide more information.

screen shot 2017-05-28 at 3 56 59 pm

Relinquishment page

 Change Benefit relinquishment to "Benefits relinquishment" (plural on Benefits)

### CODE REVIEW

File: src/js/edu-benefits/5490/config/form.js [hunk 1]
```diff
@@ -194,11 +194,11 @@ const formConfig = {
           }
         },
         benefitHistory: {
-          title: 'Benefit history',
+          title: 'Benefits history',
           path: 'benefits/history',
           initialData: {},
           uiSchema: {
-            'ui:title': 'Benefit history',
+            'ui:title': 'Benefits history',
```
Comment (cehsu) [inline src/js/edu-benefits/5490/config/form.js]: I replaced Benefit with Benefits here as well to be consistent, please let me know if this should be reverted.

File: src/js/edu-benefits/5490/components/IntroductionPage.jsx [hunk 2]
```diff
@@ -31,7 +31,7 @@ class IntroductionPage extends React.Component {
               </ul>
               <div className="usa-alert usa-alert-info">
                 <div className="usa-alert-body">
-                  <span>You won’t be able to save your work or come back to finish. So before you start, it’s a good idea to gather information about your education history and the school you want to attend.</span>
+                  <span><strong>You won’t be able to save your work or come back to finish.</strong> So before you start, it’s a good idea to gather information about your education history and the school you want to attend.</span>
                 </div>
               </div>
               <p>We encourage you to work with a trained professional who can help you pick the right program. Call <a href="tel:888-442-4551">888-442-4551</a> (888-GI-BILL-1) from 8:00 a.m. to 7:00 p.m. (ET) to talk with an education specialist</p>
```
Comment (jbalboni) [inline src/js/edu-benefits/5490/components/IntroductionPage.jsx]: Can you add a `<br/>` above this to add some spacing between the blue box and the next line? We did that on the 5495, probably should be on the rest of the intro pages, too.

File: src/js/edu-benefits/5490/config/form.js (hunk 1 above)
Comment (jbalboni) [inline src/js/edu-benefits/5490/config/form.js]: Looks good to me.

**Review by jbalboni — APPROVED** [review]
LGTM. Had one nitpick about spacing, good to merge after that.

File: src/js/edu-benefits/5490/helpers.jsx [hunk 3]
```diff
@@ -51,15 +51,15 @@ export const benefitsRelinquishedWarning = (
 );
 
 export const benefitsDisclaimerSpouse = (
-  <div className="usa-alert usa-alert-warning usa-content">
+  <div className="usa-alert usa-alert-info">
```
Comment (cehsu) [inline src/js/edu-benefits/5490/helpers.jsx]: @goldenmeanie @peggygannon I see you ultimately decided to leave this as is to make it more subdued (in your discussion in department-of-veterans-affairs/vets.gov-team#2867). I will revert.

File: src/js/edu-benefits/5490/helpers.jsx [hunk 4]
```diff
@@ -51,13 +51,17 @@ export const benefitsRelinquishedWarning = (
 );
 
 export const benefitsDisclaimerSpouse = (
-  <p>
-    IMPORTANT: If you are eligible for Chapter 35 Survivors’ and Dependents’ Educational Assistance Program (DEA) and eligible for Chapter 33 Post-9/11 GI Bill Marine Gunnery Sergeant John David Fry Scholarship (Fry Scholarship), you must relinquish entitlement to the benefit that you are not applying for (even if entitlement arises from separate events). <strong>You cannot retain eligibility for both programs simultaneously</strong>.
-  </p>
+  <div className="usa-alert usa-alert-info">
```
Comment (cehsu) [inline src/js/edu-benefits/5490/helpers.jsx]: @goldenmeanie @peggygannon I see you ultimately decided to leave this as is to make it more subdued (in your discussion in department-of-veterans-affairs/vets.gov-team#2867). I will revert.

File: src/js/edu-benefits/5490/components/IntroductionPage.jsx (hunk 2 above)
Comment (cehsu) [inline src/js/edu-benefits/5490/components/IntroductionPage.jsx]: Sounds good, change made.

**Pull request merged**